testing/kea/APKBUILD

```diff
@@ -0,0 +1,163 @@
+# Maintainer: Steve Holweg <skytep@gmail.com>
+# Contributor: Steve Holweg <skytep@gmail.com>
+# Contributor: Baptiste Jonglez <baptiste--aur at jonglez dot org>
+# Contributor: nfnty
+
+pkgname=kea
+pkgver=1.3.0
+pkgrel=0
+pkgdesc="High-performance, extensible DHCP server engine from ISC, supporting both DHCPv4 and DHCPv6"
+arch="all"
+url="http://kea.isc.org"
+license="MPL2"
+depends=""
+makedepends="botan log4cplus boost-dev postgresql-dev mariadb-dev"
+subpackages="$pkgname-doc $pkgname-dev $pkgname-admin:admin:noarch $pkgname-ctrl-agent:ctrlagent $pkgname-dhcp-ddns:dhcpddns $pkgname-dhcp4 $pkgname-dhcp6 $pkgname-keactrl:keactrl:noarch $pkgname-utils"
+
+source="https://ftp.isc.org/isc/kea/${pkgver}/kea-${pkgver}.tar.gz
+				fix-scripts-include-path.patch
+				kea-dhcp4.initd
+				kea-dhcp6.initd
+				kea-dhcp-ddns.initd
+				kea-ctrl-agent.initd"
+
+validpgpkeys="BE0E9748B718253A28BB89FFF1B11BF05CF02E57" # Internet Systems Consortium, Inc. (Signing key, 2017-2018) <codesign@isc.org>
+
+build() {
+	# set default configure flags
+	_configure="--prefix=/usr \
```

Comment:
This variable is used only once, so it'd be more clear to avoid it and just pass these flags directly to make.